model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/PersistentUserSessionProvider.java

```diff
@@ -329,7 +329,6 @@ public String getId() {
             String[] idpAliasSessionId = predicate.getBrokerUserId().split("\\.");
 
             Map<String, String> attributes = new HashMap<>();
-            attributes.put(UserModel.IDP_ALIAS, idpAliasSessionId[0]);
```

Comment:
There is no need to search for the ALIAS because we only want to lookup the user based on its "external" ID at the IdP side. A specific filter is executed to actually match the session.

Reply:
See my comment above - this is needed to handle the same federated user ID with multiple brokers.